### PR TITLE
Viz tutorial

### PR DESCRIPTION
**Pull request by fliphilipp**
Just a draft - not ready for merging

### CODE REVIEW

**Comment by fliphilipp** [comment]
@scottyhq do you know why it keeps telling me it can't find my reference to the other notebook? This seems to be why the tests are failing, but I checked and it works in my repo...

`/home/runner/work/ICESat-2-Hackweek-2023/ICESat-2-Hackweek-2023/book/tutorials/DataVisualization/OpenAltimetry_Earth_Engine.ipynb:1100006: WARNING: 'myst' reference target not found: use_examples.ipynb`

**Comment by scottyhq** [comment]
I'm guessing you need a fully specified relative path `[notebook link](./use_examples.ipynb)` 

https://jupyterbook.org/en/stable/content/references.html#reference-content-files

**Comment by scottyhq** [comment]
@fliphilipp is this ready for review or still a draft?

**Comment by fliphilipp** [comment]
> @fliphilipp is this ready for review or still a draft?

I think I will make some small changes in the notebook still, but nothing major

File: book/tutorials/DataVisualization/.gitignore
Comment (scottyhq) [inline book/tutorials/DataVisualization/.gitignore:1]: I didn't realize you could have multiple .gitignores in subfolders in a repo. neat.

**Review by scottyhq — APPROVED** [review]
nice! if you want to commit these notebooks rendered with outputs that would be okay since we're not executing them.

File: book/tutorials/DataVisualization/.gitignore
Comment (fliphilipp) [reply book/tutorials/DataVisualization/.gitignore:1]: Yeah it's quite helpful, just recently learned about it myself!

**Comment by fliphilipp** [comment]
> 



> 

Oh yeah I'll commit them with the outputs then! I'll clean up a few small things and then commit and merge!

**Pull request merged**